python/cudf/cudf/core/frame.py

```diff
@@ -238,6 +239,44 @@ def _quantiles(
         result._copy_categories(self)
         return result
 
+    def repeat(self, repeats, axis=None):
+
+        assert axis in (None, 0)
+        # new_index = self.index.repeat(repeats)
+        # cols = libcudfxx.filling.repeat(self._columns, repeats)
+        # # to preserve col names, need to get it from old _cols dict
+        # column_names = self._data.names
+        # result = DataFrame(data=dict(zip(column_names, cols)))
+        # return result.set_index(new_index)
+
+        return self._repeat(repeats)
+
+    def _repeat(self, count):
+        if is_scalar(count):
+            count = Scalar(count)
+        else:
+            count = as_column(count)
+
+        result = self.__class__._from_table(
+            libcudfxx.filling.repeat(self, count)
+        )
+
+        result._copy_categories(self)
+        return result
+
+    def _fill(self, fill_values, begin=0, end=-1, inplace=False):
+        """Identify non-missing values.
```

Comment:
The docs here look incomplete/incorrect. Either remove if not needed (i.e. internal function) or complete/fix.